.github/workflows/release.yaml

```diff
@@ -1,27 +1,64 @@
 # Attempts to perform a release when a particular tag is pushed. This uses the
-# `./scripts/release.sh`, and assumes that the CRATES_TOKEN secret has been set
-# and contains an API token with which we can publish our crates to crates.io.
+# `cargo release`, and assumes that the CRATES_TOKEN secret has been set and
+# contains an API token with which we can publish our crates to crates.io.
+#
+# The `ibc-derive` publishing process is managed manually since it's not
+# consistently published with every release.
 #
 # If release operation fails partway through due to a temporary error (e.g. the
 # crate being published depends on the crate published just prior, but the prior
 # crate isn't yet available via crates.io), one can simply rerun this workflow.
-# The release.sh script aims to be an idempotent operation, skipping the
-# publishing of crates that have already been published.
 name: Release
 
 on:
   push:
+    branches:
+      - "release/*"
```

Comment:
Why do we want to run it on `release/*` branches when we already run it on tags?

Reply:
ah. `publish-check` runs on `release/` and `publish` and `gh-release` run on tags.